### DISCUSSION TITLE
How does the model "magic" happen

### DISCUSSION THREAD

**Discussion by studiopatagon** [jeremyevans/sequel#2016] [General]
I have a DB table named **Categories** and my model is named **Category**. Everything works correctly, but how does the Model know that the correct table to add data to is **Categories** ? 

It made sense to me when I have a table called **users** and the model is **User** as it's just the plural version, but I don't get how it can figure out the **Categories** scenario. 

This brings me to my next question, if needed, can you manually specify which table the Model is associated with?

**Comment by studiopatagon** [comment]
Looking at the docs I found this:

`class Comment < Sequel::Model(:something)`

I assume if I had to explicitly tell which table I'd do class `Category < Sequel::Model(:categories)`

And why it works without explicitly specifying must be because Sequel tries to figure out based on the Model name?

**Reply by janko** [reply]
Sequel is able to pluralize the model name, and assumes that's how the table name is called. Try to find "inflector" or "inflections" module in the source code.
